python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -301,9 +343,18 @@ async def _send_chat_stream_request(
     # endregion
     # region content creation
 
+    def _settings_messages_to_prompt(self, messages: list[dict[str, Any]]) -> str:
```

Comment:
should this just be called `messages_to_prompt`? I don't see the settings being used.

Reply:
These are the input messages extracted from the settings.  Renaming to _input_messages_to_prompt.